### DISCUSSION TITLE
quickstart dashboard fails for concourse 6.2 and 6.3 docker image

### DISCUSSION THREAD

**Discussion by codertao** [concourse/concourse#5874] [Help & Support] [answered]
Following the quickstart with docker-compose, on trying to hit localhost I get gray screen, with firefox console showing:

`SyntaxError: dynamic module import is not implemented elm-setup.js:1:51`

Relevant bit of elm-setup:

```
const renderingModulePromise = import('./index.mjs');

const node = document.getElementById("elm-app-embed");
if (node === null) {
  throw "missing #elm-app-embed";
}
```

(Looks like it isn't being compiled?).

Tested with concourse/concourse:latest, concourse:6.2, and :5.5; 5.5 worked, 6.2 and latest did not.

**Comment by jamieklassen** [comment, marked as answer]
converting to a discussion per https://blog.concourse-ci.org/nobody-wants-a-stale-bot/#an-update-on-triage. What version of firefox? I use firefox and i've been messing around with almost this exact setup every day for a while now... i wonder if there's some security feature related to dynamic modules?

**Reply by codertao** [reply]
As it turns out, it was a moderately out of date version of firefox- apparently firefox isn't being auto-updated on my nix box. (dynamic import support was added ~May 2019, so I've learned something new).
